packages/grafana-data/src/themes/createTypography.ts

```diff
@@ -115,8 +115,8 @@ export function createTypography(colors: ThemeColors, typographyInput: ThemeTypo
     h4: buildVariant(fontWeightRegular, 18, 1.235, 0.25),
     h5: buildVariant(fontWeightRegular, 16, 1.334, 0),
     h6: buildVariant(fontWeightMedium, 14, 1.6, 0.15),
-    body: buildVariant(fontWeightRegular, 14, 1.5, 0.15),
-    bodySmall: buildVariant(fontWeightRegular, 12, 1.5, 0.15),
+    body: buildVariant(fontWeightRegular, 14, 1.57142857143, 0.15),
+    bodySmall: buildVariant(fontWeightRegular, 12, 1.57142857143, 0.15),
```

Comment:
maybe we could use a utility function to calculate this line height (or, buildVariant should calculate it?). This value is just begging for someone else to come along and simplify it down to 1.6 😅

Reply:
wonder whether we should just change `buildVariant` to pass a `line-height` in px (then use `pxToRem` as we do with `font-size`) instead of a ratio? 🤔

Reply:
although... https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#prefer_unitless_numbers_for_line-height_values

Reply:
although although... https://twitter.com/steveschoger/status/1253695374155530240?lang=en-GB

Reply:
I mean calculate the unit-less value though. `buildVariant` takes line height in absolute px, and then calculates the unitless line-height multiplier.

```
lineHeight: lineHeightPx / fontSize
```